### DISCUSSION TITLE
Can't sync since upgrading to 1.16

### DISCUSSION THREAD

**Discussion by mauibilly** [Chia-Network/chia-blockchain#6278] [Support]
Small time farmer with limited technology skills. I upgraded to V 1.16 on May 21. Haven't been able to synch since. I can log into my wallet, but that's it. 

Running Mojave 10.14.16 Have tried multiple restarts and disconnects...

Any suggestions?

**Comment by tiny0little** [comment]
validate inbound `tcp8444`
add introducers and node.chia.net
info on both here - [Resolving-Sync-Issues---Port-8444](https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444)

if above does not help, please post outputs of `chia version`, `chia show -c` and `chia show -s`

**Comment by room101-dev** [comment]
> Small time farmer with limited technology skills. I upgraded to V 1.16 on May 21. Haven't been able to synch since. I can log into my wallet, but that's it.
> 
> Running Mojave 10.14.16 Have tried multiple restarts and disconnects...
> 
> Any suggestions?

USE cmd line, use linux only

First 'chia start node', once that is done, then start the wallet sync ( chia show -s ) will dump sync status

Once the blockchain is synced, then run 'chia start wallet', this will also take a day, the above may take 2-3 days.
To check wallet sync status the cmd is 'chia wallet show'

to get into cmd-line mode its ". ./activate" from the chia-blockchain folder


Now your done, you can run the GUI, but its crap.

Nobody needs to run this crappy software to earn XCH, just run the plotter 'chia plots create ..', make plots, and mine them on HPOOL and earn XCH, bad news is its now 800 plots to get 1 XCH in a month. Unless you have a super-computer to plot, and have 100TB of disk, and 1/2 dozen NVME drives to sacrifice, your wasting your  time.

...

You can add the 'introducers' by cmd-line 'chia add', but its far better to edit the config.yaml (Chia file in ~/.chia/mainnet/config) and learn to do stuff your self, other wise you will be dependent upon the morons (chia-net-dev) forever.

**Reply by mauibilly** [reply]
Ha ha. Thanks. I've earned a few Chia. I was relatively early even with my crappy little farm. I'll give both of these suggestions a shot tomorrow. My wallet is synching fine. I just can't connect to mainnet and haven't been able to since 5/20. Cheers.

**Comment by BrandtH22** [comment]
@mauibilly , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
